README.md

```diff
@@ -22,11 +22,15 @@ want anything else installed system-wide.
 
 ### Option 1) Building with Bundler
 
-`cd` into the directory where you cloned this repository, then install the required gems with `bundle install`. This will automatically put the gems into `./bundle-vendor/bundle`.
-
-Start the server in the context of the bundle:
-
-    bundle exec jekyll serve
+Install ruby and rbenv. `cd` into the directory where you cloned this repository, then execute the following:
+```
+rbenv install 2.4.0
+rbenv local 2.4.0
+rbenv rehash
+gem install bundle
```

Comment:
Does the command `rbenv shell 2.4.0` do the same thing?  I never knew it existed until recently.  Works well, for me.